nilearn/surface/tests/test_surface.py

```diff
@@ -71,6 +71,18 @@ def data(self):
         return self._data
 
 
+def test_load_surf_data(tmp_path):
```

Comment:
not sure we should keep this given that we already have tests for this function

Reply:
Agreed.